utils/heatmap_dataset.py

```diff
@@ -49,6 +50,8 @@ def __init__(
         self.lower_bound = lower_bound
         self.upper_bound = upper_bound
 
+        # TODO: n_samples should be derived from how often the bounding box fits
+        # rather than the number of volumes (as in SegmentationDataset)
```

Comment:
The logic for doing this is here: https://github.com/constantinpape/torch-em/blob/main/torch_em/data/segmentation_dataset.py#L53